.github/workflows/check_website.yaml

```diff
@@ -0,0 +1,46 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Website Check
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+      - name: This workflow is under maintenance!
```

Comment:
Is this PR working yet?

Reply:
Seems to be!

https://github.com/EthanThatOneKid/acmcsuf.com/runs/4123540714?check_suite_focus=true